internal/k8s/leader_election_callbacks.go

```diff
@@ -26,6 +28,10 @@ func (c *AviController) OnStartedLeading() {
 	utils.AviLog.Debugf("AKO became a leader")
 	lib.AKOControlConfig().PodEventf(v1.EventTypeNormal, "LeaderElection", "AKO became a leader")
 	c.publishAllParentVSKeysToRestLayer()
+
+	// once the l3 cache is populated, we can call the updatestatus functions from here
+	restlayer := rest.NewRestOperations(avicache.SharedAviObjCache(), avicache.SharedAVIClients())
```

Comment:
@swathinsankara: This will be called before removing stale VS. I think this will create some issue of populating unwanted VS status. Did you check how it behaves when namespaceSync feature enabled?

Reply:
Addressed the comment.